src/targets/gpu/include/migraphx/gpu/time_op.hpp

```diff
@@ -35,6 +36,8 @@ namespace gpu {
 MIGRAPHX_GPU_EXPORT double
 time_op(context& ictx, operation op, const std::vector<shape>& inputs, int n = 100);
 
+MIGRAPHX_GPU_EXPORT double time_program(context& ictx, migraphx::program p, int n = 100);
```

Comment:
Wouldn't we want something more statistically significant like say 1k runs instead of 100?

Reply:
this would get called while benchmarking multiple candidate solutions while doing exhaustive tuning. 
There can be hundreds for applicable solutions for a single gemm or other operator. Running each candidate 1k time would make it a lot slower. 

100 is just a default parameter.  While benchmarking it simply uses 20. 

Doing 1k runs may also have adverse effect if GPU becomes hot and it may throttle.

Reply:
understood